README.md

```diff
@@ -1,5 +1,5 @@
-# https://github.com/eyoab01/Microverse-project-mobfinish
-#  https://github.com/eyoab01/mobile-portfolio-project-final
+# [https://eyoab01.github.io/mobile-portfolio-project-final/]
+# [https://eyoab01.github.io/mobile-portfolio-project-final/]
```

Comment:
- [ OPTIONAL ] I recommend using the microverse [README](https://github.com/microverseinc/readme-template) template since it is very organized and it has all the required sections to help you showcase your project💪🏻💯🔥